src/components/UniversalTable/types.tsx

```diff
@@ -1,8 +1,8 @@
-export type BaseData = {
+export interface BaseData {
```

Comment:
I believe you can extend a type as well, so guess no need to add this change.

Reply:
OK, i`ll check it.

Reply:
i`ve returned prev staete of this row

Reply:
+